src/Common/src/Interop/Windows/winhttp/Interop.winhttp_types.cs

```diff
@@ -310,5 +310,13 @@ public struct WINHTTP_WEB_SOCKET_STATUS
             public uint dwBytesTransferred;
             public WINHTTP_WEB_SOCKET_BUFFER_TYPE eBufferType;
         }
+
+        [Flags]
+        internal enum AutoDetectType
```

Comment:
nit: 'internal' should be 'public' like all the other nested classes/types in this class.  The enclosing class is already 'internal'

Reply:
Will change.